pom.xml

```diff
@@ -593,6 +594,21 @@
       <artifactId>proto</artifactId>
       <version>${tensorflow.version}</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-secretmanager</artifactId>
```

Comment:
Added `libraries-bom` v20.0.0 for secret-manager.

.
Had to explicitly add `proto-google-cloud-secretmanager-v1` to the dependency list because `maven-dependency-plugin` complains about the dependency being used but not declared and causes `mvn clean install` to fail.
```
[WARNING] Used undeclared dependencies found:
[WARNING]    com.google.api.grpc:proto-google-cloud-secretmanager-v1:jar:1.4.2:compile
```
I've removed it from the dependency list now and set `maven-dependency-plugin` to ignore the warning.

Reply:
Could you please follow the convention of declaring the versions in the properties list? So something like:

```
<cloud-libraries-bom.version>20.0.0</cloud-libraries-bom.version>
...
      <version>${cloud-libraries-bom.version}</version>
```

If nothing else, it's more consistent with how we manage versions for other dependencies.

Also, could you remove the comment about this being for secret manager? While that is the purpose for now, `libraries-bom` covers a lot more of Google Cloud's artifacts. It'll come in handy in other cases sooner or later.

Reply:
Sure thing. 

Added a property for the `libraries-bom` version and removed the comment. Please check again.

Reply:
Thanks!